tests/services/org.freedesktop.Flatpak.SystemHelper.service.in

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Name=org.freedesktop.Flatpak.SystemHelper
+Exec=@libexecdir@/flatpak-system-helper --session --no-idle-exit
```

Comment:
Might be worth adding a comment here to point out that `--session` is added deliberately to make it work with our mock D-Bus session bus.

Reply:
done